packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -120,17 +128,16 @@ class VsCode {
     final String progFiles86 = platform.environment['programfiles(x86)'];
     final String progFiles = platform.environment['programfiles'];
 
-    final Map<String, String> stable = <String, String>{
-      fs.path.join(progFiles86, 'Microsoft VS Code'): '.vscode',
-      fs.path.join(progFiles, 'Microsoft VS Code'): '.vscode'
-    };
-    final Map<String, String> insiders = <String, String>{
-      fs.path.join(progFiles86, 'Microsoft VS Code Insiders'):
-          '.vscode-insiders',
-      fs.path.join(progFiles, 'Microsoft VS Code Insiders'): '.vscode-insiders'
-    };
-
-    return _findInstalled(stable, insiders);
+    return _findInstalled(<_VsCodeInstallLocation>[
+      new _VsCodeInstallLocation(fs.path.join(progFiles86, 'Microsoft VS Code'), '.vscode',
+          edition: '32-bit edition'),
+      new _VsCodeInstallLocation(fs.path.join(progFiles, 'Microsoft VS Code'), '.vscode',
+          edition: '64-bit edition'),
+      new _VsCodeInstallLocation(fs.path.join(progFiles86 , 'Microsoft VS Code Insiders'), '.vscode',
```

Comment:
Not `.vscode-insiders` as it was before?

Reply:
Oh boy, that was a copy/paste error. Thanks! I guess having real integration tests that spin up Windows VMs with VS Code installed in various combinations is out of scope?

(Fixed!)